protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1181,6 +1187,9 @@ message ObserveAccountsRequest {
   string asset = 3;
   // Account type to subscribe to, required field.
   vega.AccountType type = 4;
+  // Whether to return all derived parties from AMMs for the given party.
+  // If used, party ID is required.
+  optional bool include_derived_parties = 5;
```

Comment:
These new fields probably also need to be included in GraphQL

Reply:
You are right. Just added it.

Reply:
@karlem Should the parent party ID field go in too, or is it already there? I didn't see it in the changes.

Reply:
Maybe not. let me double check

Reply:
Should be theere.